### DISCUSSION TITLE
Firefox release

### DISCUSSION THREAD

**Discussion by scratchusernamemrtbts** [ScratchAddons/ScratchAddons#338] [Help] [answered]
when will you release it to firefox
and also i think you need to remove the link to firefox in the scratchaddon.com first or someone will get confused when they clicked it and see a massaging extension instead of the addon

**Comment by Hans5958** [comment, marked as answer]
Yes, we aware about this. I'm also the one that keep waiting for this. Hopefully @WorldLanguages and @griffpatch sorted this out.

Semi-duplicate of #324.

**Comment by WorldLanguages** [comment]
Moving to #324
